install/helm/agones/templates/service/allocation.yaml

```diff
@@ -276,3 +279,24 @@ data:
 {{- else }}
   tls-ca.crt: {{ .Files.Get "certs/allocator/server.crt" | b64enc }}
 {{- end }}
+
+---
+# Default allocation client secret
+{{- if .Values.agones.allocator.generateClientTLS }}
```

Comment:
Also update the doc: https://github.com/googleforgames/agones/blob/master/site/content/en/docs/Installation/Install%20Agones/helm.md

Reply:
Done.